### DISCUSSION TITLE
Adding border radius to course links on the landing page

### DISCUSSION THREAD

**Discussion by VenkaSri** [TheOdinProject/theodinproject#4142] [Ideas]
What are your thoughts on adding a border radius to the links on the landing page? This would also help maintain consistency with the other links on the page. I have attached screenshots of both styles for your reference.

Without border radius:
<img width="300" alt="Screenshot 2023-09-15 at 9 38 04 PM" src="https://github.com/TheOdinProject/theodinproject/assets/53845532/52693663-d6bf-459c-a089-55cd83adb098">

With border radius:
<img width="300" alt="Screenshot 2023-09-15 at 9 37 52 PM" src="https://github.com/TheOdinProject/theodinproject/assets/53845532/e094b217-1b83-498d-ab83-8e234c647f40">

**Comment by ManonLef** [comment]
I do like it. I believe it would best be made consistent with the team page updates that are being implemented when it comes to amount of radius. What do you think @KevinMulhern ?

Also noticing that React icon looks too yellow

**Reply by KevinMulhern** [reply]
I concur, sounds like a good idea 💪 

Re: The React icon - we have someone on the case https://github.com/TheOdinProject/theodinproject/issues/4088.

**Reply by ManonLef** [reply]
Found it when I wanted to open an issue :D

**Reply by VenkaSri** [reply]
@KevinMulhern Should I open an issue to work on making that change?

**Reply by KevinMulhern** [reply]
@VenkaSri with this being such a small change, it would be worth skipping making an issue and just going straight to PR. It's worth mentioning the border-radius Manon mentioned we're using for the new team page is `rounded-2xl` [from Tailwind](https://tailwindcss.com/docs/border-radius).

**Reply by VenkaSri** [reply]
@ManonLef @KevinMulhern Do you guys see `rounded-2xl` being applied for member card in new team page? 


<img width="1242" alt="Screenshot 2023-09-17 at 8 58 51 PM" src="https://github.com/TheOdinProject/theodinproject/assets/53845532/8ac35791-d7dd-4641-a61e-c505741bf0b1">

**Reply by ManonLef** [reply]
@VenkaSri you can see it on the latest PR deployment :) https://odin-review-app-pr-4125.herokuapp.com/team
